src/test/java/uk/sky/cqlmigrate/SchemaUpdatesTest.java

```diff
@@ -97,20 +85,19 @@ public void schemaUpdatesTableShouldNotBeCreatedIfExists() throws Exception {
             fail("Expected " + SCHEMA_UPDATES_TABLE + " table creation to be attempted only once.");
         }
         //then
-        KeyspaceMetadata keyspaceMetadata = cluster.getMetadata().getKeyspace(TEST_KEYSPACE);
+        KeyspaceMetadata keyspaceMetadata = session.getMetadata().getKeyspace(TEST_KEYSPACE).get();
         assertThat(keyspaceMetadata.getTable(SCHEMA_UPDATES_TABLE)).as("table should have been created").isNotNull();
```

Comment:
can check optional is empty? if the test fails the .get() with throw an exception rather than fail this assertion.